File: ZooJava/src/main/java/com/example/zoojava/ZooApplication.java
package com.example.zoojava;

import com.example.zoojava.managers.SceneManager;
import com.example.zoojava.utils.DataSystem;
import javafx.application.Application;
import javafx.stage.Stage;

import java.io.IOException;

/**
 * @author Rocío P.F
 * @version 1.0
 * github: Rochiio
 * Programa para la administración de un zoológico, haciendo test.
 */
public class ZooApplication extends Application {
//TODO al final de hacer el programa añadir logger

    @Override
    public void start(Stage stage) throws IOException {
        SceneManager.loginScene(stage);
    }

    public static void main(String[] args) {
        DataSystem system = new DataSystem();
        system.addData();
        launch();
        system.backupAndClean();
    }



}

File: ZooKotlin/src/main/kotlin/com/example/zookotlin/ZooApplication.kt
package com.example.zookotlin

import com.example.zookotlin.managers.SceneManager
import com.example.zookotlin.utils.DataSystem
import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Scene
import javafx.stage.Stage

/**
 * @author Rocío P.F
 * @version 1.0
 * github: Rochiio
 * Programa para la administración de un zoológico, haciendo test.
 */
class ZooApplication : Application() {
    override fun start(stage: Stage) {
       SceneManager.loginScene(stage)
    }
}

fun main() {
    val system = DataSystem()
    system.addData()
    Application.launch(ZooApplication::class.java)
    system.backupAndClean()
}